bin/i18n/resources/apps/labs.rb

```diff
@@ -1,41 +1,23 @@
-require 'fileutils'
-require 'json'
-
 require_relative '../../i18n_script_utils'
-require_relative '../../redact_restore_utils'
+
+Dir[File.expand_path('../labs/**/*.rb', __FILE__)].sort.each {|file| require file}
 
 module I18n
   module Resources
     module Apps
       module Labs
-        I18N_SOURCE_DIR_PATH = CDO.dir(File.join(I18N_SOURCE_DIR, 'blockly-mooc')).freeze
+        DIR_NAME = 'blockly-mooc'.freeze
+        I18N_SOURCE_DIR_PATH = CDO.dir(I18N_SOURCE_DIR, DIR_NAME).freeze
+        UNTRANSLATABLE = %w[calc eval netsim].freeze
+        REDACTABLE = %w[applab gamelab weblab].freeze
```

Comment:
Not blocking, but I wonder if it will be convenient to use more contextual constant names like `REDACTABLE_LABS` or `UNTRANSLATABLE_LABS` if we are going to have redactable/untranslatable in other parts of the sync.
I see the potential benefit, but also how it could be unnecessary.

Reply:
`UNTRANSLATABLE_LABS` and `REDACTABLE_LABS` were the initial name, but in the global context, it looks like `I18n::Resources::Apps::Labs::REDACTABLE_LABS` and having `labs` twice in the row looks kind of redundant, so I decided to remove it.
But you are right, `UNTRANSLATABLE_LABS` and `REDACTABLE_LABS` contain more context inside the module, will return this.